README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
```

Comment:
While not part of the immediate changes, I noticed that the README ends abruptly after the FOSSA status badge. Consider adding more sections to improve the overall structure and content of the README.
```suggestion
## License

This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.

## Acknowledgments

* Hat tip to anyone whose code was used
* Inspiration
* etc
```